src/main/index.js

```diff
@@ -28,23 +29,35 @@ const createPage = () => (
         <p>{"Hello!"}</p>
     </div>);
 
+/**
+ * Creates the Contact Us page.
+ *
+ * @function
+ */
+const createContactPage = () => (
```

Comment:
These functions that return JSX can be used as elements. Perhaps it'd be better to give these names that match the element they're creating (complete with Capitalization).

`const ContactPage = () => (<div>...</div>)` and then later: `<ContactPage />`

Reply:
I attempted this, but I wasn't able to get it working as described. We can have an offline discussion about this.